src/System.Text.Json/tests/Json.cs

```diff
@@ -87,32 +204,19 @@ private string OutputObject(Object obj)
         {
             var str = "";
 
-            if (obj == null || obj.Members == null) return str;
-
-            foreach (var member in obj.Members)
-            {
-                str += "{";
-                str += OutputMembers(member);
-                str += "}";
-            }
-
-            return str;
-        }
-
-        private string OutputMembers(Members members)
-        {
-            var str = "";
-
-            if (members == null || members.Pairs == null) return str;
+            if (obj == null || obj.Pairs == null) return str;
 
-            for (var i = 0; i < members.Pairs.Count; i++)
+            str += "{";
```

Comment:
Usually StringBuilders give better performance when manipulating strings in a loop